### PR TITLE
[KOGITO-9683] - Update Operator's installation guide to reflect OLM

### PR DESCRIPTION
**Pull request by ricardozanini**
**JIRA:** https://issues.redhat.com/browse/KOGITO-9683

**Description:**
Small update to the Operator Installation Guide to point users to OLM/OperatorHub's way of installing. The old installation method now is only for "local/manual" use cases, as highlighted in the document.

Please make sure that your PR meets the following requirements:

- [x] You have read the [contributions doc](https://github.com/kiegroup/kogito-docs/blob/main/CONTRIBUTING.md)
- [x] Pull Request title is properly formatted: `KOGITO-XYZ Subject`
- [x] Pull Request title contains the target branch if not targeting main: `[0.9.x] KOGITO-XYZ Subject`
- [x] The nav.adoc file has a link to this guide in the proper category
- [x] The index.adoc file has a card to this guide in the proper category, with a meaningful description

<details>
<summary>
How to backport a pull request to a different branch?
</summary>

In order to automatically create a **backporting pull request** please add one or more labels having the following format `backport-<branch-name>`, where `<branch-name>` is the name of the branch where the pull request must be backported to (e.g., `backport-7.67.x` to backport the original PR to the `7.67.x` branch).

> **NOTE**: **backporting** is an action aiming to move a change (usually a commit) from a branch (usually the main one) to another one, which is generally referring to a still maintained release branch. Keeping it simple: it is about to move a specific change or a set of them from one branch to another.

Once the original pull request is successfully merged, the automated action will create one backporting pull request per each label (with the previous format) that has been added.

If something goes wrong, the author will be notified and at this point a manual backporting is needed.

> **NOTE**: this automated backporting is triggered whenever a pull request on `main` branch is labeled or closed, but both conditions must be satisfied to get the new PR created.
</details>

### CODE REVIEW

File: serverlessworkflow/modules/ROOT/pages/cloud/operator/install-serverless-operator.adoc [hunk 1]
```diff
@@ -3,17 +3,53 @@
...
+
+This guide describes how to install the {operator_name} in a Kubernetes or OpenShift cluster. The operator is in an xref:/cloud/operator/known-issues.adoc[early development stage] (community only) and has been tested on OpenShift 4.11+, Kubernetes 1.22+, and link:{minikube_url}[Minikube].
 
 .Prerequisites
-* A Kubernetes cluster with admin privileges. Alternatively, you can use Minikube or KIND.
+* A Kubernetes or OpenShift cluster with admin privileges. Alternatively, you can use Minikube or KIND.
 * `kubectl` command-line tool is installed. Otherwise, Minikube provides it.
 
-== Prepare a Minikube Instance
+== {product_name} Operator OpenShift Installation
```
Comment (kaldesai) [inline serverlessworkflow/modules/ROOT/pages/cloud/operator/install-serverless-operator.adoc:20]:
```suggestion
== {product_name} Operator OpenShift installation
```

File: serverlessworkflow/modules/ROOT/pages/cloud/operator/install-serverless-operator.adoc [hunk 2]
```diff
@@ -3,17 +3,53 @@
...
+
+To install the operator on OpenShift follow the document "link:{openshift_operator_install_url}[Adding Operators to a cluster]" from the OpenShift's documentation.
```
Comment (kaldesai) [inline serverlessworkflow/modules/ROOT/pages/cloud/operator/install-serverless-operator.adoc:22]:
```suggestion
To install the operator on OpenShift refer to the "link:{openshift_operator_install_url}[Adding Operators to a cluster]" from the OpenShift's documentation.
```

File: serverlessworkflow/modules/ROOT/pages/cloud/operator/install-serverless-operator.adoc [hunk 3]
```diff
@@ -3,17 +3,53 @@
...
+
+When searching for the operator in the *Filter by keyword* field, use the word `{operator_openshift_keyword}`. If you're installing from the CLI, the operator's catalog name is `{operator_openshift_catalog}`.
+
+=== Uninstall
+
+To remove the operator on OpenShift follow the document "link:{openshift_operator_uninstall_url}[Deleting Operators from a cluster]" from the OpenShift's documentation.
```
Comment (kaldesai) [inline serverlessworkflow/modules/ROOT/pages/cloud/operator/install-serverless-operator.adoc:28]:
```suggestion
To remove the operator on OpenShift refer to the "link:{openshift_operator_uninstall_url}[Deleting Operators from a cluster]" from the OpenShift's documentation.
```

File: serverlessworkflow/modules/ROOT/pages/cloud/operator/install-serverless-operator.adoc [hunk 4]
```diff
@@ -3,17 +3,53 @@
...
+When searching for the subscription to remove, use the word `{operator_k8s_subscription}`.
+
+== {product_name} Operator Manual Installation
 
 [WARNING]
 ====
+If you're running on Kubernetes or OpenShift, It's highly recommended to install the operator from the OperatorHub or OpenShift Console instead since the installation is managed by OLM. Use this method only if you need a snapshot version or you're running locally on Minikube or KIND.
+====
+
+=== Prepare a Minikube Instance
```
Comment (kaldesai) [inline serverlessworkflow/modules/ROOT/pages/cloud/operator/install-serverless-operator.adoc:49]:
```suggestion
=== Prepare a Minikube instance
```

**Review by kaldesai — APPROVED** [review]
@ricardozanini Minor suggestion! Otherwise LGTM!

File: serverlessworkflow/modules/ROOT/pages/cloud/operator/install-serverless-operator.adoc [hunk 5]
```diff
@@ -3,17 +3,53 @@
...
+
+To remove the operator on Kubernetes follow the document "link:{kubernetes_operator_uninstall_url}[Uninstall your operator]" from the OLM's documentation.
+
+When searching for the subscription to remove, use the word `{operator_k8s_subscription}`.
+
+== {product_name} Operator Manual Installation
 
 [WARNING]
 ====
+If you're running on Kubernetes or OpenShift, It's highly recommended to install the operator from the OperatorHub or OpenShift Console instead since the installation is managed by OLM. Use this method only if you need a snapshot version or you're running locally on Minikube or KIND.
```
Comment (radtriste) [inline serverlessworkflow/modules/ROOT/pages/cloud/operator/install-serverless-operator.adoc:46]:
```suggestion
If you're running on Kubernetes or OpenShift, it is highly recommended to install the operator from the OperatorHub or OpenShift Console instead since the installation is managed by OLM. Use this method only if you need a snapshot version or you're running locally on Minikube or KIND.
```

**Review by radtriste — APPROVED** [review]
minor comment

File: serverlessworkflow/modules/ROOT/pages/cloud/operator/install-serverless-operator.adoc [hunk 6]
```diff
@@ -41,11 +77,8 @@ If Minikube does not work with the default driver, also known as `docker`, you c
 minikube start [...] --driver podman
 ----
 
-== Install the {product_name} Operator
```
Comment (krisv) [inline serverlessworkflow/modules/ROOT/pages/cloud/operator/install-serverless-operator.adoc:44]: Since you mention in the note that the section to install minikube can be skipped, I think we should make sure the installation of the operator is in a separate section (otherwise people don't know how far they need to skip), so I suggest keeping this line, with === title.

File: serverlessworkflow/modules/ROOT/pages/cloud/operator/install-serverless-operator.adoc [hunk 7]
```diff
@@ -92,8 +125,7 @@ sonataflowplatforms.sonataflow.org   2023-03-08T18:31:15Z
 sonataflows.sonataflow.org   2023-03-08T18:31:15Z
```
Comment (krisv) [inline serverlessworkflow/modules/ROOT/pages/cloud/operator/install-serverless-operator.adoc:125]: This section on operator CRDs, is this necessary?  The wording is a bit confusing to me ("you must first install" gives the impression the user still needs to do something), and by this time the operator should have done that already?

**Review by krisv — APPROVED** [review]

File: serverlessworkflow/modules/ROOT/pages/cloud/operator/install-serverless-operator.adoc (hunk 7 above)
Comment (ricardozanini) [reply serverlessworkflow/modules/ROOT/pages/cloud/operator/install-serverless-operator.adoc:125]: Yes, the YAML applied before should've applied the CRDs. Let me rephrase this section.
